specification/e-referrals-service-api.yaml

```diff
@@ -502,16 +504,20 @@ paths:
         - find services that meet the referral needs of a patient using [[HYPERLINK_A010]], and
         - create a referral using [[HYPERLINK_A011]] or [[HYPERLINK_A021]]
         
-        You may also wish to present the user-friendly names for the reference data, for example when viewing the referral using [[HYPERLINK_A005]].
+        You may also wish to present the user-friendly names for the reference data, for example when viewing the referral using [[HYPERLINK_A005]] 
+        and [[HYPERLINK_A038]].
```

Comment:
Not sure why but this is not showing the link for A038 in the named spec

Reply:
I'll find out what to do with this - get appointment (038) is not on this branch